pulp_container/app/registry_api.py

```diff
@@ -169,6 +177,24 @@ def __init__(self, digest):
         )
 
 
+class ManifestSignatureInvalid(ParseError):
+    """An exception to render an HTTP 400 response with the code 'SIGNATURE_INVALID'."""
+
+    def __init__(self, digest):
+        """Initialize the exception with the digest of a signed manifest."""
+        super().__init__(
+            detail={
+                "errors": [
+                    {
+                        "code": "SIGNATURE_INVALID",
+                        "message": "signature invalid",
+                        "detail": {"digest": digest},
```

Comment:
how about manifest_digest, at first i thought this is signature digest until i looked further into the code.